spec/models/institute_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Institute, type: :model do
+  
+  subject { institute }
+  context 'with valid attributes' do
+    let!(:institute) { build(:institute, name: "institute1") }
+    
+    it { is_expected.to validate_presence_of(:name) }
+
+    it "is valid" do
+      subject
+      expect(institute).to be_valid
+    end
+  end
+
+  context "with invalid attributes" do
+      let!(:institute) { build(:institute, name: nil) }
+
+      it "is not valid" do
+        institute.valid?
+        expect(institute.errors[:name]).to include("can't be blank")
+      end
+  end
+
```

Comment:
dont need this empty line.

Both these contexts in this spec could be cleaned up.
By adding an extra let value for name .. you only have the one build line.
Also the build is only building the object .. not saving it .. so you do not need the bang !
```
  subject { institute }
  let(:institute) { build(:institute, name: name) }

  context 'with valid attributes' do
    let(:name) { "institute1" }

    it { is_expected.to validate_presence_of(:name) }
    it { is_expected.to be_valid }
  end

  context 'with invalid attributes' do
    let(:name) { nil }

    it { is_expected.to_not be_valid }
  end
```